docs/internal.md

```diff
@@ -29,14 +30,14 @@ yarn dev
 
 1. Go to [actions page for "Packages the internal version"](https://github.com/KILTprotocol/sporran-extension/actions/workflows/publish.yaml).
 1. Click on the build you want to test.
-1. Download the extension file from the _Artifacts_ section at the bottom.
-1. Unpack the downloaded _.zip_ file and unpack the _sporran-???.zip_ from it as well.
+1. Download the extension file from the _Artifact's_ section at the bottom.
```

Comment:
Since the section is called "Artifacts" I think the grammar checker misguided you here. I have my doubts about any apostrophe here, but I am ready to accept the version consistent with the one you have below.
```suggestion
1. Download the extension file from the _Artifacts’_ section at the bottom.
```

Reply:
I'm just deleting them all. English grammar is not known for being strict.

Reply:
gone